librz/main/rz-find.c

```diff
@@ -157,7 +160,9 @@ static int show_help(const char *argv0, int line) {
 		" -n         do not stop on read errors\n"
 		" -r         print using rizin commands\n"
 		" -s [str]   search for a specific string (can be used multiple times)\n"
-		" -S [str]   search for a specific wide string (can be used multiple times). Assumes str is UTF-8.\n"
+		" -u [str]   search for a specific wide string (can be used multiple times). Assumes str is UTF-8.\n"
```

Comment:
I wonder whether `w` would make more sense... for example, in rizin there is `/w` to search for wide strings.

Reply:
Changed to `w`. Thanks for your review.